packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -576,6 +618,92 @@ module('Integration | operator-mode', function (hooks) {
     assert.dom('[data-test-stack-card-index="1"]').includesText('Mango');
   });
 
+  test('it allows chat commands to change cards in the stack', async function (assert) {
```

Comment:
you might wanna think about using the new onSave() callbacks to assert that cards are not saved during previews...